netbox_netprod_importer/__main__.py

```diff
@@ -62,8 +62,10 @@ def parse_args():
             dest="overwrite", action="store_true"
         )
         sp.add_argument(
-            "-d", "--debug", help="enable debug, verbose output",
-            dest="debug", action="store_true"
+            "-v", "--verbose", metavar="LEVEL",
+            help="enable debug or warning, verbose output",
+            dest="verbose",
+            choices=['warning', 'debug']
```

Comment:
Could you change your simple quotes to double quotes please? Just to be coherent with your PR and the rest of the code.

Reply:
Removed because transferred and changed processing